src/renderer/page/settings/view.jsx

```diff
@@ -265,6 +272,13 @@ class SettingsPage extends React.PureComponent<Props, State> {
                 checked={autoplay}
                 postfix={__('Autoplay media files')}
               />
+              <FormField
+                type="checkbox"
+                name="auto_download"
+                onChange={this.onAutoDownloadChange}
+                checked={autoDownload}
+                postfix={__('Automatically download new content')}
```

Comment:
This should reference that it is for subscriptions. 
`Automatically download new content from your subscriptions`? That might be a little long